### PR TITLE
DAC6-3048: create your-fis page

### PR DESCRIPTION
**Pull request by oarshad**
Updated /your-fis page

### CODE REVIEW

File: app/views/YourFinancialInstitutionsView.scala.html [hunk 1]
```diff
@@ -16,12 +16,35 @@
...
 
-@()(implicit request: Request[_], messages: Messages)
+@(form: Form[_], institutions: SummaryList)(implicit request: Request[_], messages: Messages)
 
-@layout(pageTitle = titleNoForm(messages("yourFinancialInstitutions.title"))) {
-   <!--TODO: This is a placeholder page. Will be implemented as part of DAC6-3048-->
-    <h1 class="govuk-heading-xl">@messages("yourFinancialInstitutions.heading")</h1>
+@layout(pageTitle = title(form, messages("yourFinancialInstitutions.title"))) {
+    @if(institutions.rows.isEmpty) {
+        <h1 class="govuk-heading-xl">@messages("yourFinancialInstitutions.heading.noFinancialInstitutions")</h1>
```
Comment (C-Thomp) [inline app/views/YourFinancialInstitutionsView.scala.html:29]: minor - we can use the Heading component to tidy the html up here

File: app/views/templates/Layout.scala.html [hunk 2]
```diff
@@ -56,6 +56,7 @@
     )
 
     <link href="@routes.Assets.versioned("stylesheets/application.css")" media="all" rel="stylesheet" type="text/css" />
+    <link href="@routes.Assets.versioned("crs-fatca-fi-management.css")" media="all" rel="stylesheet" type="text/css" />
```
Comment (C-Thomp) [inline app/views/templates/Layout.scala.html:59]: what does adding the width: inherited do?

Comment (C-Thomp) [reply app/views/templates/Layout.scala.html:59]: ah i think i see

**Review by C-Thomp — DISMISSED** [review]

**Review by koladeadewuyi — DISMISSED** [review]

File: app/views/YourFinancialInstitutionsView.scala.html (hunk 1 above)
Comment (oarshad) [reply app/views/YourFinancialInstitutionsView.scala.html:29]: sorry @C-Thomp , I have to leave early so I couldn't get a chance to update it. As I am away next week, please feel free to make any updates you like

File: app/views/components/Heading.scala.html [hunk 3]
```diff
@@ -16,6 +16,6 @@
 
 @this()
 
-@(headingContent: String, size: String = "l")
+@(headingContent: String, size: String = "l", extraClasses: String = "")
```
Comment (C-Thomp) [inline app/views/components/Heading.scala.html:19]: 👍

**Review by C-Thomp — APPROVED** [review]
All great! ⚡

**Pull request merged**